packages/devtools/web/styles.css

```diff
@@ -799,3 +824,69 @@ html [full] {
 span.strong {
     font-weight: 600;
 }
+
+/*
+ * Support tagging elements with collapsible that can have children tagged
+ * with optional-text to be hidden when the page is below a certain threshold.
+ *
+ * Also support optional directly on elements to be hidden.
+ */
+
+@media all and (max-width: 550px) { .optional-550 { display: none; } }
+@media all and (max-width: 700px) { .optional-700 { display: none; } }
```

Comment:
nit: maybe put all the CSS that depend on a specific max-width together in the same media query. So add the optional-700 and collapsible-700 rules to the same media query.

Reply:
I had it that way originally, but it was harder to glance in the CSS to see which widths we already had classes for that you could use, and which you needed to add new ones for (it kinds bugs me that we have to add a rule for each one, but I don't know there's a better way).

WDYT?

Reply:
Go ahead and leave it as is.